ArduCopter/mode_follow.cpp

```diff
@@ -114,8 +114,8 @@ void ModeFollow::run()
 
             case AP_Follow::YAW_BEHAVE_DIR_OF_FLIGHT: {
                 // Face the direction of travel
-                if (vel_ofs_ne_cms.length_squared() > (100.0 * 100.0)) {
-                    yaw_rad = vel_ofs_ne_cms.angle();
+                if (vel_ofs_ne_ms.length_squared() > (100.0 * 100.0)) {
```

Comment:
I guess we should change the right side to be just 1?

Reply:
yes, good catch.